server-side/email-service/app/consumer.js

```diff
@@ -0,0 +1,34 @@
+const kafka = require('kafka-node');
+const nodemailer = require('nodemailer');
+const Bottleneck = require('bottleneck');
+const retry = require('async-retry');
+const { sendEmail } = require('./mailer');
```

Comment:
use import instead of require wherever it possible (internal exports change to es6 syntax if needed)